src/rs.cpp

```diff
@@ -84,9 +84,39 @@ struct rs2_device_list
     std::vector< std::shared_ptr< librealsense::device_info > > list;
 };
 
+struct rs2_option_value_wrapper : rs2_option_value
+{
+    // Keep the original json value, so we can refer to it (e.g., with as_string)
+    std::shared_ptr< rsutils::json > p_json;
+
+    // Add a reference count to control lifetime
+    mutable std::atomic< int > ref_count;
```

Comment:
Who increases this count above 1?

Reply:
Any time you get the value multiple times.
There should be an increase in `rs2_get_option_value_from_list` but there isn't, I'll fix.